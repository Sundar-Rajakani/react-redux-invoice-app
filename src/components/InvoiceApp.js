import React from 'react';

import Invoices from './Invoices';
import InvoicesSummary from './InvoicesSummary'
import InvoiceFilters from './InvoiceFilters';

const InvoiceApp = () => (
  <div>
    <InvoicesSummary />
    <InvoiceFilters />
    <Invoices />
  </div>
);

export default InvoiceApp;