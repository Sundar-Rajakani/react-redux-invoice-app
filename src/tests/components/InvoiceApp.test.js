import React from 'react';
import { shallow } from 'enzyme';
import InvoiceApp from '../../components/InvoiceApp';

test('should render Invoice App page ', () => {
  const wrapper = shallow(<InvoiceApp />);
  expect(wrapper).toMatchSnapshot();
});