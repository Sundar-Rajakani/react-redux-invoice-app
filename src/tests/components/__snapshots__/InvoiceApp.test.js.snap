// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Invoice App page  1`] = `
<div>
  <Connect(InvoicesSummary) />
  <Connect(InvoiceFilters) />
  <Connect(Invoices) />
</div>
`;
